apps/nextjs/src/components/layout/header.tsx

```diff
@@ -24,9 +25,10 @@ export const MainHeader = ({ logo, actions, hasNavigation = true }: Props) => {
           <UnstyledButton component={Link} href="/">
             {logo ?? <HomarrLogoWithTitle size="md" />}
           </UnstyledButton>
+          <NavigationBar />
         </Group>
-        <DesktopSearchInput />
         <Group h="100%" align="center" justify="end" style={{ flex: 1 }} wrap="nowrap">
+          <DesktopSearchInput />
```

Comment:
The goal was to still have the search input at the same position when no navigation items are selected